doc/radosgw/role.rst

```diff
@@ -70,18 +72,23 @@ Request Parameters
 :Description: Name of the role.
 :Type: String
 
-For example:: 	
+For example:
+
+.. prompt:: bash
 	
-  radosgw-admin role delete --role-name=S3Access1
+   radosgw-admin role delete --role-name=S3Access1
 
-Note: A role can be deleted only when it doesn't have any permission policy attached to it.
+Note: A role can be deleted only when it has no permission policy attached to
+it.
```

Comment:
We shouldn't have to, but maybe add a note that if any policies are present they must be deleted first?

Reply:
Accepted. I'm willing to belabor the obvious if it helps readers to be sure of their situations.

Reply:
We've seen the value of doing so quiet recently on the list